### PR TITLE
fellowships homepage wireframe

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #950

WIP Comps: https://projects.invisionapp.com/share/ZJ9YTAGEA#/screens/273312987

This PR adds elements to the fellowships homepage. I didn't wanna bother adding CSS since the comps haven't been finalized and Sketch file is not ready yet. We'll fine tune stylings at a later stage.

### CODE REVIEW

File: network-api/networkapi/templates/foundation-base.html [hunk 1]
```diff
@@ -72,6 +72,34 @@
         </div>
       </div>
     </div>
+    {% if showHeroBanner %}
```
Comment (ScottDowne) [inline network-api/networkapi/templates/foundation-base.html]: you can change this if to use blocks.

Like this:

```
{% block hero_banner %}{% endblock %}
```

You can also fill in that block with a default banner unrelated to fellows, but I don't think we need to in our case.

Then in fellows_home.html somewhere you can do:

```
{% block hero_banner %}
    <div id="hero">
      <div class="container">
        <div class="row">
          <div class="col-md-12 col-lg-6 py-5">
            <div class="mb-4">
              <h1 class="h1-white">Fellowships</h1>
            </div>
            <p class="lead-black">Our Fellowships empower bright minds in the fields of public policy, science and journalism.</p>
          </div>
        </div>
      </div>
    </div>
{% endblock %}
```

and then remove the `, { 'showHeroBanner': True }` in fellows/views.py

Comment (ScottDowne) [reply network-api/networkapi/templates/foundation-base.html]: I think. I've  not tested this...

Comment (mmmavis) [reply network-api/networkapi/templates/foundation-base.html]: Sounds good. Will fix!

File: network-api/networkapi/templates/foundation-base.html [hunk 2]
```diff
@@ -72,6 +72,34 @@
...
+            <div class="mb-4">
+              <h1 class="h1-white">Fellowships</h1>
+            </div>
+            <p class="lead-black">Our Fellowships empower bright minds in the fields of public policy, science and journalism.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+    <div id="fellowships-nav" class="mb-5">
```
Comment (ScottDowne) [inline network-api/networkapi/templates/foundation-base.html]: Is the nav supposed to be available to all pages extending foundation-base.html?

Comment (mmmavis) [reply network-api/networkapi/templates/foundation-base.html]: From what I see on https://projects.invisionapp.com/d/main#/console/10016110/263404839/preview ... yea I think the fellowship nav is always there.

It seems like the homepage is the only page that has the hero banner though. Did I understand this correctly, @taisdesouzalessa ?

File: network-api/networkapi/fellows/templates/fellows_home.html [hunk 3]
```diff
@@ -1,5 +1,161 @@
...
+    <a href="" class="cta-link">Learn More</a>
+  </div>
+</div>
+
+<div class="row mb-5">
+  <div class="col-md-12 my-5">
+    <hr class="hr-gradient mt-0">
+    <h4 class="h2-headings-white">Meet Some of the Mozilla Fellows</h4>
+  </div>
+  <div class="featured-person col-6 col-sm-4 col-md-3 mb-4 mb-sm-0" key={index}>
```
Comment (ScottDowne) [inline network-api/networkapi/fellows/templates/fellows_home.html]: What's the index from?

File: network-api/networkapi/fellows/templates/fellows_home.html [hunk 4]
```diff
@@ -1,5 +1,161 @@
...
+      <li>Sugar plum caramels jelly.</li>
+      <li>Marshmallow jelly beans sesame snaps fruitcake jelly-o icing bear claw dessert.</li>
+    </ul>
+  </div>
+  <div class="col-md-6">img</div>
+</div>
+
+<div class="row py-5 px-md-2">
+  <div class="col-md-2">img</div>
+  <div class="col-md-10">
```
Comment (ScottDowne) [inline network-api/networkapi/fellows/templates/fellows_home.html]: A lot of chunks in the body block can probably be rendered from an inclusion tag (kinda like setting up a component)

Although these last two chunks of html are almost the same cept the top one (block this comment in line attached to) has `col-md-2` then `col-md-10` and the last one just under this comment has `col-md-1` and `col-md-11` which is the type of minor variance that makes things like an inclusion tag a bit annoying.

Comment (ScottDowne) [reply network-api/networkapi/fellows/templates/fellows_home.html]: I'm also not sure if any of these inclusion tags would be used outside of this page or not, or just this page, and if so, does jdnago have a way to do it in a light way way.

File: network-api/networkapi/fellows/templates/fellows_home.html (hunk 3 above)
Comment (mmmavis) [reply network-api/networkapi/fellows/templates/fellows_home.html]: ha this was some copy and paste mistake

File: network-api/networkapi/utility/templatetags/active_nav.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
+from django import template
```
Comment (mmmavis) [inline network-api/networkapi/utility/templatetags/active_nav.py]: copied from https://maorleger.wordpress.com/2015/11/20/active-page-styling-for-bootstrap-navbar-and-django/

should I leave a comment here and cite that URL?

**Comment by mmmavis** [comment]
@alanmoo I think all other fellowships pages will inherit from `foundation-base.html` so I left fellowships nav code there and not `fellows_home.html`.

File: network-api/networkapi/utility/templatetags/active_nav.py (hunk 5 above)
Comment (alanmoo) [reply network-api/networkapi/utility/templatetags/active_nav.py]: Probably useful for future reference.

**Comment by mmmavis** [comment]
🎉

**Pull request merged**